src/dialogs/more-info/controls/more-info-script.ts

```diff
@@ -148,7 +148,15 @@ class MoreInfoScript extends LitElement {
     const newState = this.stateObj;
 
     if (newState && (!oldState || oldState.entity_id !== newState.entity_id)) {
-      this._scriptData = { service: newState.entity_id, data: {} };
+      const fields =
+        this.hass.services.script[computeObjectId(newState.entity_id)]?.fields;
+      const defaultValues = {};
```

Comment:
I think we should use [computeInitialHaFormData](https://github.com/home-assistant/frontend/blob/505d7b6ddba7f10eaf9cfd30fd677f04dc90258a/src/components/ha-form/compute-initial-ha-form-data.ts#L4) here.